docker-for-mac/index.md

```diff
@@ -131,6 +133,16 @@ You can set the following runtime options.
   Uncheck this to opt out and prevent auto-send of data. Docker may prompt for
   more information in some cases, even with auto-send enabled.
 
+#### Choose whether to store Docker credentials on Mac keychain (Edge feature)
```

Comment:
@mstanleyjones Okay I tried that include and it weirdly had no effect (didn't show up). I'm going to let this finish building, and merge to get it published, but I'll experiment with those this afternoon. I have some other stuff to fix as well, later. Thanks!